next/app/test/belle/projectmodal/page.tsx

```diff
@@ -0,0 +1,3 @@
+import ProjectModal from "@/components/projects/ProjectModal";
+
+export default function ProjectModal
```

Comment:
P.s. this file currently breaks the app if I go to `http://localhost:3000/test/belle/projectmodal`, small fix but important to have it fixed cause I think it'll break the deployment.

Reply:
I have deleted all code from this file, hopefully it fixes it